js/renovation/ui/grids/data_grid_next/widgets/data_cell.tsx

```diff
@@ -67,19 +67,20 @@ export class DataCellProps {
   defaultOptionRules: null,
   view: viewFunction,
 })
-export class DataCell extends JSXComponent(DataCellProps) {
+export class DataCell extends JSXComponent<DataCellProps, 'column' | 'row'>(DataCellProps) {
   get cellText(): string {
-    const { dataField } = this.props.column;
-    const value = dataField && this.props.row.data[dataField];
-    return value !== undefined ? String(value) : '';
+    const { format, calculateCellValue } = this.props.column;
+    const value = calculateCellValue ? calculateCellValue(this.props.row.data) : undefined;
+    return formatHelper.format(value, format) || '';
   }
 
   get classes(): string {
-    const { columnIndex, countColumn } = this.props;
+    const { columnIndex, columnCount, column } = this.props;
 
     const classesMap = {
       [CLASSES.firstChild]: columnIndex === 0,
-      [CLASSES.lastChild]: columnIndex === countColumn - 1,
+      [CLASSES.lastChild]: columnIndex === columnCount - 1,
```

Comment:
remove `lastChild`/`firstChild` classes, fix css to use `:last-of-type` instead if `last-child`

Reply:
and then remove `columnIndex`, `columnCount`